tests/taskpoller.go

```diff
@@ -172,6 +172,7 @@ Loop:
 		if response.Query == nil || !opts.PollSticky {
 			// if not query task, should have some history events
 			// for non sticky query, there should be events returned
+			// read all the history events into events slice
```

Comment:
tbh i think this comment is too much like the code (assuming it's referring to `events = history.Events`